webboot/webboot.go

```diff
@@ -8,9 +8,26 @@
 package main
 
 import (
-	"fmt"
+	"flag"
+	"log"
+
+	"github.com/u-root/webboot/pkg/dhclient"
+)
+
+var (
+	ifName  = flag.String("InterfaceName", "^e.*", "Name of the interface")
+	timeout = flag.Int("timeout", 15, "Lease timeout in seconds")
+	retry   = flag.Int("retry", 5, "Max number of attempts for DHCP clients to send requests. -1 means infinity")
+	verbose = flag.Bool("verbose", false, "Verbose output")
+	ipv4    = flag.Bool("ipv4", true, "use IPV4")
+	ipv6    = flag.Bool("ipv6", true, "use IPV6")
 )
 
 func main() {
-	fmt.Println("weboot up")
+	//	flag.Parse()
+	//	if len(flag.Args()) > 1 {
+	//		log.Fatalf("Only one agument supported. Got %v, want name of an ISO", flag.Args())
+	//	}
```

Comment:
either remove this comment block or use it for something.

Reply:
i removed the commented area. I believed I needed it to handle the "webboot [nameOfOS]" or "webboot [URL]" but then realized it was not necessary at the point for dhclient, so I commeted it out. But the comment is removed from the code as it is not necessary for dhclient. I will check for this flag length in the next PR